cmd/helm/template_test.go

```diff
@@ -149,6 +149,7 @@ func TestTemplateCmd(t *testing.T) {
 			out := bytes.NewBuffer(nil)
 			cmd := newTemplateCmd(out)
 			cmd.SetArgs(tt.args)
+			cmd.SetOutput(out)
```

Comment:
I'm wondering if the `cmd.SetOutput(out)`call  should be part of `newTemplateCmd`?

Specifically, I see `newTemplateCmd` takes `out` as input, and assigns it to `templateCmd.out`. But then as far as I can tell, `templateCmd.out` is never used. So maybe the fact that we need `cmd.SetOutput` indicates `newTemplateCmd` is not functioning as we expect.

Reply:
Thank you for the review. Yes, it seems the template command does not utilize the `io.Writer` it takes as input

 If fixing this on the template command is desired please let me know and I will submit a PR.

Reply:
Thats a great question, and one I'm not actually positive of the answer to. My gut says yes fixing this on the template command is desired, but I don't have enough experience to say for certain.